package.json

```diff
@@ -13,21 +13,22 @@
   },
   "devDependencies": {
     "chai": "^4.0.2",
+    "cross-env": "^5.0.5",
```

Comment:
This is just a question, I understand this is to provide a cross-compatible env, but afaik, envs also work in windows. What is the case we need this?

Reply:
When I run `npm run start-dev` without `cross-env` I get the following error:
`'NODE_ENV' is not recognized as an internal or external command,`

Reply:
out of curiosity, What OS are you using?

I agree is better to use cross-env to avoid this kind of problems

Reply:
Windows 7